cogs/msg.py

```diff
@@ -76,12 +174,14 @@ async def on_message(self, message):
 
         await message.channel.send(result[0][1])
 
-
```

Comment:
This newline is mandated by some PEP guideline (at least the last time that I ran pylint), as it separates the class from other non-class-related functions.